yw_website/yw_db/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+
+# Create your models here.
+
+class Tag(models.Model):
+    parent_tag = models.ForeignKey('self', blank=True, on_delete=models.DO_NOTHING, default=None)
+
+    WORKFLOW = "w"
+    VERSION = "v"
+    RUN = "r"
+
+    TAG_CHOICES = (
```

Comment:
there is no tag choice for file but you seemed to want to add tags to `File` table